site/videosync.html

```diff
@@ -15,6 +15,7 @@
 		<div id="player_container"></div>
 		<div id="bottom">
 			<div id="video_controls_bar">
+        <button id="remove_all_previous">Remove Previous Videos</button>

```

Comment:
Sure thing. my editor is set to 2 spaces instead of a tab, as most community style guides use that standard